.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
#### Critical
Please create a more comprehensive `.gitignore` file. Check [here](https://www.npmjs.com/package/gitignore)

**NB**
- Please install `gitignore` as a global dependency on your system. It shouldn't be part of the list of dependencies for the project